protocol-units/bridge/move-modules/sources/atomic_bridge_counterparty.move

```diff
@@ -107,6 +108,17 @@ module atomic_bridge::atomic_bridge_counterparty {
         )
     }
 
+    #[view]
+    public fun get_bridge_transfer_state(bridge_transfer_id: vector<u8>): u8 acquires BridgeTransferStore, BridgeConfig {
```

Comment:
why adding a specific get bridge transfer state function? why not just extract it from the tuple? Is this mostly just for testing or is bridge_transfers unable to return data?

Reply:
was for testing; i got bridge_transfers returning data now